class.jetpack-modules-list-table.php

```diff
@@ -309,6 +311,16 @@ function column_default( $item, $column_name ) {
 		}
 	}
 
+	//Check if the info parameter provided in the URL corresponds to an actual module
+	function module_info_check( $info = false, $modules ) {
+		if ( false == $info ) {
+			return false;
+		}
+		else if ( array_key_exists( $info, $modules ) ) {
```

Comment:
Nitpick: let's keep this `else if` on the same line as the closing brace of the previous block.